lib/diagnostic.rb

```diff
@@ -78,17 +93,24 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = I am a HouseCat, and I go 'meow'
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+# HouseCat points to Cat which points to Animal which prints the sentence which gets the HouseCat name and teh HouseCat sound
 ##
 # Define a new class, 'Lion', which (a) inherits from 'Cat',
 # (b) uses the 'Carnivorous' module below as a mixin, and
 # (c) adds a new method called `roar`, which prints out "ROAR!"
 
+class Lion < Cat
+  def initialize
```

Comment:
This constructor will attempt to assign an instance variable `@roar`, to a local variable or parameter named `roar`, which doesn't exist, and cause an error.